crates/diesel_models/src/merchant_connector_account.rs

```diff
@@ -47,6 +58,65 @@ pub struct MerchantConnectorAccount {
     pub connector_wallets_details: Option<Encryption>,
 }
 
+#[cfg(all(
+    any(feature = "v1", feature = "v2"),
+    not(feature = "merchant_connector_account_v2")
+))]
+impl MerchantConnectorAccount {
+    pub fn get_id(&self) -> String {
+        self.merchant_connector_id.clone()
+    }
+}
+
+#[cfg(all(feature = "v2", feature = "merchant_connector_account_v2"))]
+#[derive(
+    Clone,
+    Debug,
+    serde::Serialize,
+    serde::Deserialize,
+    Identifiable,
+    Queryable,
+    Selectable,
+    router_derive::DebugAsDisplay,
+)]
+#[diesel(table_name = merchant_connector_account, check_for_backend(diesel::pg::Pg))]
+pub struct MerchantConnectorAccount {
+    pub merchant_id: id_type::MerchantId,
+    pub connector_name: String,
+    pub connector_account_details: Encryption,
+    pub disabled: Option<bool>,
+    #[diesel(deserialize_as = super::OptionalDieselArray<serde_json::Value>)]
+    pub payment_methods_enabled: Option<Vec<serde_json::Value>>,
+    pub connector_type: storage_enums::ConnectorType,
+    pub metadata: Option<pii::SecretSerdeValue>,
+    pub connector_label: Option<String>,
+    pub frm_configs: Option<Secret<serde_json::Value>>,
+    pub created_at: time::PrimitiveDateTime,
+    pub modified_at: time::PrimitiveDateTime,
+    pub connector_webhook_details: Option<pii::SecretSerdeValue>,
+    #[diesel(deserialize_as = super::OptionalDieselArray<pii::SecretSerdeValue>)]
+    pub frm_config: Option<Vec<Secret<serde_json::Value>>>,
```

Comment:
```suggestion
    pub frm_config: Option<Vec<Secret<pii::SecretSerdeValue>>>,
```

Reply:
There are two frm configs we should remove one which is deprecated.

Reply:
Address all these in other related types too (like `MerchantConnectorAccountNew`, update types, update internal types)